README.md

```diff
@@ -23,14 +23,14 @@ Getting started with the OpenHIM-core
 
 To make use of your own custom configurations you can copy the [default.json](https://github.com/jembi/openhim-core-js/blob/master/config/default.json) config file and override the default setting:
 
-* Please specify the timezone's offset value otherwise the offset used will be that of the server's timezone.  You can do this by changing the value of the utcOffset in the default.json.
-
 ```
 wget https://raw.githubusercontent.com/jembi/openhim-core-js/master/config/default.json
 # edit default.json, then
 openhim-core --conf=path/to/default.json
 ```
 
+> To specify the timezone that the openhim services are used in, change the utcOffset value in the default.json file. If the timezone's offset value is not specified, the dafault offset of +200 will be used. This ensures that weekly and daily channel reports have the correct timestamps. 
```

Comment:
@bradsawadye Technically untrue - the default offset will only give the correct channel reports if the users are in that time zone :p (IE: Us). 
If we default the time to +2 we should say something along the lines of that the majority of known users are in South Africa.
But to be more generic +0 should be the default as I would think that would be the expected function.

Also, what happened with the discussion about the "moment timezone" package that would account for Daylight savings?